cypress/integration/indices_spec.js

```diff
@@ -24,6 +24,9 @@ describe("Indices", () => {
     // Set welcome screen tracking to false
     localStorage.setItem("home:welcome:show", "false");
 
+    // Wait for 5s to avoid 503 server unavailable error
+    cy.wait(5000);
```

Comment:
Instead of a sleep, would it be possible for us to leverage [wait-on](https://docs.cypress.io/guides/guides/continuous-integration.html#Solutions) in the workflow itself to wait for the server to be available?

This would be more flexible in the cases in which the time to be available is variable and possibly longer than expected.

Reply:
The explicitly "wait" is my only concern. Although it can be used, make sure there is no better ways.

Reply:
I agree that having an specified wait condition would be better. Will look into the usage of wait-on

Reply:
The updated "wait on" in the workflow looks cool. 👍

Reply:
Removed `cy.wait(5000)` in `indices_spec.js`, `policies_spec.js` and `managed_indices_spec.js` files. Using `wait-on` in cypress workflow file.